R/exclude.R

```diff
@@ -98,17 +178,21 @@ normalize_exclusions <- function(x, normalize_path = TRUE,
 
   # no named parameters at all
   if (is.null(names(x))) {
-    x <- structure(relist(rep(Inf, length(x)), x), names = x)
+    # x is a character vector of file names
+    # Normalize to list(<filename> = list(Inf), ...)
+    x <- structure(rep(list(Inf), length(x)), names = x)
```

Comment:
I think `setNames(rep(.), x)` looks nicer & IINM is faster

Reply:
TIL `setNames` is pretty fast

```r
microbenchmark::microbenchmark(
  structure(rep(list(Inf), length(letters)), names = letters),
  setNames(rep(list(Inf), length(letters)), letters)
)
## Unit: microseconds
##                                                         expr min  lq  mean median  uq  max neval
##  structure(rep(list(Inf), length(letters)), names = letters) 3.8 4.0 5.977   4.15 4.4 55.5   100
##           setNames(rep(list(Inf), length(letters)), letters) 1.3 1.4 1.832   1.60 1.7 22.1   100
```

Reply:
I think rather, IME, `structure` is pretty slow